### ISSUE TITLE
program is stuck on the "Calibrating..." page. 

### ISSUE THREAD

**Issue by icyqwq** [me-no-dev/ESP32Lepton#1]
Hello, I am trying to implement this code on my M5Stack Core. But my program is stuck on the "Calibrating..." page. I used a logic analyzer to capture the waveform of the VOSPI and it seems that the FLIR is working properly. I have used a jumper to connect FLIR's VOSPI to a different SPI bus than the Core's screen. In addition, I am using the Lepton 3.0 module, there is no telemetry function. So I set the Lepton mode to LEPTON_FORMAT_RAW in the program.  But it still doesn't work, do you have any ideas?

**Comment by me-no-dev** [comment]
huh? the example as-is on M5Stack with their FLIR module and Lepton 3.0 inside (telemetry and radiometry are different things). And is running great. Same SPI as the LCD and all. No change, just compile and flash. Same code is running on a custom pin setup where I use different SPIs for Lepton and LCD and it's again working as expected. Did you at all get this lepton module to work with other code/device?

**Comment by icyqwq** [comment]
Yes, it works fine on the M5 module. But the frame rate is not good, I am trying to optimize performance.

**Comment by me-no-dev** [comment]
frame rate is limited by the manufacturer to 9 FPS. There is no way around it :)

**Comment by sergey1492** [comment]
Hi, I uploaded your program to my m5stack, but I also got it on calibration, what can I try to do? Also, some squeaks are heard from the speaker during operation, is this normal? Module Lepton 3.5.
